README.md

```diff
@@ -24,3 +24,504 @@ You can also learn more about our [Governance](/GOVERNANCE.md) structure.
+            "system:serviceaccount:ack-system:ack-applicationautoscaling-controller"
+          ]
+        }
+      }
+    }
+  ]
+}
+' > ./trust.json
+```
+
+
+Run the following command to create a role with the trust relationship defined in `trust.json`. This role enables the Amazon EKS cluster to get and refresh credentials from IAM.
+```sh
+OIDC_ROLE_NAME=ack-sage-role-$CLUSTER_NAME
+
+aws --region $AWS_DEFAULT_REGION iam create-role --role-name $OIDC_ROLE_NAME --assume-role-policy-document file://trust.json
+
+# Attach the AmazonSageMakerFullAccess Policy to the Role
+aws --region $AWS_DEFAULT_REGION iam attach-role-policy --role-name $OIDC_ROLE_NAME --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
+export OIDC_ROLE_ARN=$(aws --region $AWS_DEFAULT_REGION iam get-role --role-name $OIDC_ROLE_NAME --output text --query 'Role.Arn')
+echo $OIDC_ROLE_ARN
+```
+
+Take note of OIDC_ROLE_ARN printed in the previous step; you will pass this value to the service account used by the controller.
+
+### 3.0 Install Controllers
+
+#### 3.1 Install SageMaker Controller
+
+##### 3.1.1 Download helm chart
+
+```sh
+export HELM_EXPERIMENTAL_OCI=1
+export SERVICE=sagemaker
+export RELEASE_VERSION=v0.0.1
+export CHART_EXPORT_PATH=/tmp/chart
+export CHART_REPO=public.ecr.aws/aws-controllers-k8s/chart
+export CHART_REF=$CHART_REPO:$SERVICE-$RELEASE_VERSION
+
+mkdir -p $CHART_EXPORT_PATH
+helm chart pull $CHART_REF
+helm chart list
+helm chart export $CHART_REF --destination $CHART_EXPORT_PATH
+```
+
+##### 3.1.2 Choose one of the two options for deployment
+
+  - [Option 1] Cluster scoped deployment
+    - ```sh
+      # Change values in helm chart
+      vim $CHART_EXPORT_PATH/ack-$SERVICE-controller/values.yaml
+
+      # line #29, #30, change the aws.region, aws.account_id
+      aws:
+        region: "<REGION_HERE>"
+        account_id: "<ACCOUNT_ID_HERE>"
+
+      # line #46, remove {} in serviceAccount.annotations
+      # line #47, uncomment eks.amazonaws.com/role-arn and add the OIDC_ROLE_ARN
+      annotations: <REMOVE {}>
+      eks.amazonaws.com/role-arn: <OIDC_ROLE_ARN_HERE>
+      ```
+  - [Option 2] Namespace scoped deployment
+    - ```sh
+      # Change values in helm chart
+      vim $CHART_EXPORT_PATH/ack-$SERVICE-controller/values.yaml
+
+      # line #29, #30, change the aws.region, aws.account_id
+      aws:
+        region: "<REGION_HERE>"
+        account_id: "<ACCOUNT_ID_HERE>"
+
+      # Change the watchNamespace parameter
+      # line #33
+      watchNamespace: <Namespace_To_Listen_To>
+
+      # line #46, remove {} in serviceAccount.annotations
+      # line #47, uncomment eks.amazonaws.com/role-arn and add the OIDC_ROLE_ARN
+      annotations: <REMOVE {}>
+          eks.amazonaws.com/role-arn: <OIDC_ROLE_ARN_HERE>
+      ```
+##### 3.1.3 Install Controller
+
+Create a namespace
+```sh
+export ACK_K8S_NAMESPACE=ack-system
+kubectl create namespace $ACK_K8S_NAMESPACE
```

Comment:
Helm provides a `--create-namespace` option that would make this step unnecessary.

Reply:
good one!